server/middleware/validation.js

```diff
@@ -16,7 +17,10 @@ const validationErrors = {
     isValidWithinDate: label => `${label} must be within the last ${VALID_DAYS_RANGE} days`,
     validCaseReference: () => 'Case reference is not valid',
     contributionsFulfilled: () => 'Case contributions have to be completed or cancelled',
-    oneOf: () => 'Options are not valid'
+    oneOf: () => 'Options are not valid',
+    isValidMonth: label =>  `${label} must contain a valid month`,
+    isYearWithinRange: label =>   `${label} must contain a valid year`,
+    isValidDay: label => `${label} must contain a valid day`,
```

Comment:
GOV.UK Design System just goes for "X must be a real date" and doesn't be more specific (as long as only the invalid day/month/year field gets highlighted in red). 

"Real" is probably understood better than "valid" by normal users.

Reply:
Due to the shared use of the validation code along with the factories to create forms by everything in the front end editing it to match the guidance exactly appears too high risk/effort to be valuable so deviating slightly from the defined wording to get a similar effect remained the only option. Agreed that "Real" will likely carry more meaning for people so reworded there.